packages/zoslogs/README.md

```diff
@@ -0,0 +1,45 @@
+# z/OS Console Package
+
+Contains APIs to interact with logs on z/OS (using z/OSMF log REST endpoints).
+z/OSMF version 2.4 or higher is required. Ensure that the [z/OSMF Operations Log Support is available via APAR and associated PTFs](https://www.ibm.com/support/pages/apar/PH35930).
+## API Examples
+
+**List z/OS logs**
+
+```typescript
+import { IProfile, Session, Logger, LoggingConfigurer, ImperativeError, CredentialManagerFactory } from "@zowe/imperative";
+import { ZosmfSession } from "@zowe/zosmf-for-zowe-sdk";
+import { getDefaultProfile } from "@zowe/core-for-zowe-sdk";
+import { GetZosLog, IZosLogParms, IZosLogType } from "@zowe/zos-logs-for-zowe-sdk";
+
+(async () => {
+  //Initialize the Imperative Credential Manager Factory and Logger
+  Logger.initLogger(LoggingConfigurer.configureLogger("lib", { name: "test" }));
+  // Uncommment the below line if the Secure Credential Store is in use
+  // await CredentialManagerFactory.initialize({service: "Zowe-Plugin"});
+
+  // Get the default z/OSMF profile and create a z/OSMF session with it
+  let defaultZosmfProfile: IProfile;
+  try {
+    defaultZosmfProfile = await getDefaultProfile("zosmf", true);
+  } catch (err) {
+    throw new ImperativeError({ msg: "Failed to get a profile." });
+  }
+
+  const session: Session = ZosmfSession.createBasicZosmfSession(defaultZosmfProfile);
```

Comment:
In @next createBasicZosmfSession does not exist, so the example in the readme will not work for consumers.